src/panoptes/pocs/focuser/astromechanics.py

```diff
@@ -1,7 +1,5 @@
-from panoptes.utils import error
 from panoptes.utils.serial.device import find_serial_port
 from panoptes.pocs.focuser.serial import AbstractSerialFocuser
-from panoptes.pocs.utils.logger import get_logger
```

Comment:
As above, this is being removed because `get_logger` is not used?

Reply:
Same thing, it's not being used.